.travis.yml

```diff
@@ -32,20 +33,21 @@ before_install:
     chmod +x docker-compose
     sudo mv docker-compose /usr/local/bin
 
-install: ./mvnw install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -q -T 2
+install: ./mvnw install -DskipTests=true $MAVEN_SKIP_CHECKS_AND_DOCS -B -T 2 -q -pl '!presto-docs,!presto-server-rpm'
 
 script:
```

Comment:
It's piggy-backed on the integration test row, see [here](https://github.com/prestodb/presto/pull/5421/commits/311c3ab6e9854f8cd8fac57011222982e555c814#diff-354f30a63fb0907d4ad57269548329e3R50)